test/server.test.ts

```diff
@@ -1,17 +1,17 @@
-const request = require('supertest')
-const {createServer} = require('../src/server')
-const {logger} = require('../src/logger')
+import request from 'supertest'
+import { logger } from '../src/logger'
+import { createServer } from '../src/server'
 
-describe('server', function () {
-  let server
-  let webhook
+describe('server', () => {
+  let server: any
+  let webhook: any
```

Comment:
Instead of `any`, could we use `express.Application` for both of these?

Reply:
Hi,
On using express.Application for webhook, it gives an error like this:

 error TS2322: Type 'Mock<any>' is not assignable to type 'Application'.
  Property 'init' is missing in type 'Mock<any>'.

I googled it but couldn't understand. Can you guide me through this?

Reply:
Ahh, it's because on line 10 below, webhook is being assigned a mock value. Maybe try changing that line to:

```js
    webhook = jest.fn((req, res, next) => next()) as express.Application
```

Reply:
Hi,
I changed line 10 as you said but it still gives the same error on that line.

Reply:
I poked around with this for a while and was running into the same errors. Turns out that mocking with Jest+TypeScript is a little more involved than I imagined. http://jonathancreamer.com/testing-typescript-classes-with-jest-and-jest-mocks/.